src/main/java/org/opentestsystem/shared/search/persistence/SearchableRepositoryImpl.java

```diff
@@ -14,15 +14,19 @@
 
 import org.opentestsystem.shared.search.domain.AbstractSearchRequest;
 import org.opentestsystem.shared.search.domain.SearchResponse;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.mongodb.core.MongoOperations;
 import org.springframework.data.mongodb.core.query.Query;
 import org.springframework.data.mongodb.repository.query.MongoEntityInformation;
 import org.springframework.data.mongodb.repository.support.SimpleMongoRepository;
+import org.springframework.stereotype.Component;
 
+@Component
```

Comment:
Why did the Spring upgrade require the addition of this annotation?  Is there an underlying change that I should look at to see why this was necessary?